gulpfile.babel.js

```diff
@@ -88,4 +89,11 @@ gulp.task('bootstrap_test', () => {
     .once('end', () => {
 	process.exit();	
     })
-});
\ No newline at end of file
+});
+
+// FIXME(jimmy): This command can not remove node_modules directory (need to fix it)
```

Comment:
I'm not sure but this line looks longer than 80bytes?
If so, please add a line break.